diff -up chromium-90.0.4430.72/chrome/browser/component_updater/registration.cc.widevine-no-download chromium-90.0.4430.72/chrome/browser/component_updater/registration.cc
--- chromium-90.0.4430.72/chrome/browser/component_updater/registration.cc.widevine-no-download	2021-04-16 15:49:21.010870111 -0400
+++ chromium-90.0.4430.72/chrome/browser/component_updater/registration.cc	2021-04-16 15:57:12.153555913 -0400
@@ -96,10 +96,6 @@ void RegisterComponentsForUpdate(bool is
   // TODO(crbug.com/1069814): Remove after 2021-10-01.
   CleanUpPepperFlashComponent(profile_path);
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-  RegisterWidevineCdmComponent(cus);
-#endif  // BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-
 #if BUILDFLAG(ENABLE_NACL) && !defined(OS_ANDROID)
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   // PNaCl on Chrome OS is on rootfs and there is no need to download it. But
