diff -up chromium-88.0.4324.182/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc.stdmaxfix chromium-88.0.4324.182/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc
--- chromium-88.0.4324.182/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc.stdmaxfix	2021-02-17 13:16:27.120283969 -0500
+++ chromium-88.0.4324.182/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc	2021-02-17 13:17:37.951617295 -0500
@@ -135,7 +135,7 @@ static bool SetupAlternateStackOnce() {
 #else
   const size_t page_mask = sysconf(_SC_PAGESIZE) - 1;
 #endif
-  size_t stack_size = (std::max(SIGSTKSZ, 65536) + page_mask) & ~page_mask;
+  size_t stack_size = (std::max(SIGSTKSZ, static_cast<long>(65536)) + page_mask) & ~page_mask;
 #if defined(ABSL_HAVE_ADDRESS_SANITIZER) || \
     defined(ABSL_HAVE_MEMORY_SANITIZER) || defined(ABSL_HAVE_THREAD_SANITIZER)
   // Account for sanitizer instrumentation requiring additional stack space.
diff -up chromium-88.0.4324.182/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc.stdmaxfix chromium-88.0.4324.182/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc
--- chromium-88.0.4324.182/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc.stdmaxfix	2021-02-17 14:39:04.556382532 -0500
+++ chromium-88.0.4324.182/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc	2021-02-17 14:39:34.002519173 -0500
@@ -138,7 +138,7 @@ void InstallAlternateStackLocked() {
   // SIGSTKSZ may be too small to prevent the signal handlers from overrunning
   // the alternative stack. Ensure that the size of the alternative stack is
   // large enough.
-  static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
+  static const unsigned kSigStackSize = std::max(static_cast<long>(16384), SIGSTKSZ);
 
   // Only set an alternative stack if there isn't already one, or if the current
   // one is too small.
