diff -up chromium-86.0.4240.75/chrome/common/chrome_paths.cc.widevine-other-locations chromium-86.0.4240.75/chrome/common/chrome_paths.cc
--- chromium-86.0.4240.75/chrome/common/chrome_paths.cc.widevine-other-locations	2020-10-14 14:47:36.000823668 -0400
+++ chromium-86.0.4240.75/chrome/common/chrome_paths.cc	2020-10-14 14:54:49.347207638 -0400
@@ -379,6 +379,16 @@ bool PathProvider(int key, base::FilePat
 #if (defined(OS_LINUX) || defined(OS_CHROMEOS)) && \
     BUILDFLAG(BUNDLE_WIDEVINE_CDM)
     case chrome::DIR_BUNDLED_WIDEVINE_CDM:
+      base::PathService::Get(base::DIR_HOME, &cur);
+      cur = cur.Append(FILE_PATH_LITERAL(".local/lib/libwidevinecdm.so"));
+      if (base::PathExists(cur)) {
+        break;
+      }
+      // Yes, this has an arch hardcoded in the path, but at this time, it is the only place to find libwidevinecdm.so
+      if (base::PathExists(base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so")))) {
+        cur = base::FilePath(FILE_PATH_LITERAL("/opt/google/chrome/WidevineCdm/_platform_specific/linux_x64/libwidevinecdm.so"));
+        break;
+      }
       if (!GetComponentDirectory(&cur))
         return false;
 #if !defined(OS_CHROMEOS)
