diff -up chromium-87.0.4280.66/base/rand_util_posix.cc.el7 chromium-87.0.4280.66/base/rand_util_posix.cc
--- chromium-87.0.4280.66/base/rand_util_posix.cc.el7	2020-11-20 11:11:22.169119495 -0500
+++ chromium-87.0.4280.66/base/rand_util_posix.cc	2020-11-20 11:13:30.375614387 -0500
@@ -21,7 +21,8 @@
 #include "third_party/lss/linux_syscall_support.h"
 #endif
 
-#if !defined(OS_IOS) && !defined(OS_NACL)
+//EL-7 is too old to have this header
+#if 0
 // TODO(crbug.com/995996): Waiting for this header to appear in the iOS SDK.
 // (See below.)
 #include <sys/random.h>
