diff -up chromium-94.0.4606.54/remoting/host/remote_open_url_client.cc.remoting-nodestructor-fix chromium-94.0.4606.54/remoting/host/remote_open_url_client.cc
diff -up chromium-94.0.4606.54/remoting/host/remote_open_url_client_delegate_linux.cc.remoting-nodestructor-fix chromium-94.0.4606.54/remoting/host/remote_open_url_client_delegate_linux.cc
--- chromium-94.0.4606.54/remoting/host/remote_open_url_client_delegate_linux.cc.remoting-nodestructor-fix	2021-09-24 11:02:02.662082902 -0400
+++ chromium-94.0.4606.54/remoting/host/remote_open_url_client_delegate_linux.cc	2021-09-24 11:06:05.207472179 -0400
@@ -47,7 +47,7 @@ void ShowMessageDialog(const std::string
 }
 
 bool IsBrowserValid(const std::string& browser) {
-  static const base::NoDestructor<base::flat_set<std::string>> invalid_browsers(
+  static const base::NoDestructor<base::flat_set<std::string>> invalid_browsers{
       {
           // This is the chromoting forwarder itself.
           "crd-url-forwarder.desktop",
@@ -55,7 +55,7 @@ bool IsBrowserValid(const std::string& b
           // XFCE's forwarder. May potentially launch the chromoting forwarder
           // recursively.
           "xfce4-web-browser.desktop",
-      });
+      }};
   if (browser.empty()) {
     return false;
   }
