--- a/chrome/app/resources/manpage.1.in
+++ b/chrome/app/resources/manpage.1.in
@@ -3,7 +3,7 @@
 .TH @@PACKAGE@@ 1 "" "" "USER COMMANDS"
 
 .SH NAME
-@@PACKAGE@@ \- the web browser from Google
+@@PACKAGE@@ \- @@SUMMARY@@
 
 .SH SYNOPSIS
 .B @@PACKAGE@@
@@ -24,7 +24,7 @@ stable flags.
 \fB\-\-user\-data\-dir\fR=\fIDIR\fR
 Specifies the directory that user data (your "profile") is kept in.
 Defaults to
-.I $HOME/.config/@@PACKAGE@@ .
+.I $HOME/.config/@@XDG_SUBDIR@@ .
 Separate instances of @@MENUNAME@@ must use separate user data directories;
 repeated invocations of @@PACKAGE@@ will reuse an existing process for
 a given user data directory.
@@ -150,11 +150,11 @@ Comma separated list of hosts or pattern
 
 .SH FILES
 .TP
-.I $HOME/.config/@@PACKAGE@@
+.I $HOME/.config/@@XDG_SUBDIR@@
 Default directory for configuration data.
 
 .TP
-.I $HOME/.cache/@@PACKAGE@@
+.I $HOME/.cache/@@XDG_SUBDIR@@
 Default directory for cache data.  (Why?  See
 <http://standards.freedesktop.org/basedir-spec/latest/> .)
 
