diff --git a/chrome/browser/commerce/coupons/coupon_db.h b/chrome/browser/commerce/coupons/coupon_db.h
--- a/chrome/browser/commerce/coupons/coupon_db.h
+++ b/chrome/browser/commerce/coupons/coupon_db.h
@@ -5,6 +5,7 @@
 #ifndef CHROME_BROWSER_COMMERCE_COUPONS_COUPON_DB_H_
 #define CHROME_BROWSER_COMMERCE_COUPONS_COUPON_DB_H_

+#include <vector>
 #include "base/callback_helpers.h"
 #include "base/memory/weak_ptr.h"
 #include "url/gurl.h"
