diff -up chromium-92.0.4515.107/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py.py2 chromium-92.0.4515.107/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py
--- chromium-92.0.4515.107/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py.py2	2021-07-19 14:47:19.000000000 -0400
+++ chromium-92.0.4515.107/third_party/catapult/common/py_vulcanize/py_vulcanize/generate.py	2021-07-26 17:02:23.160750472 -0400
@@ -83,7 +83,7 @@ def _MinifyJS(input_js):
 
   with tempfile.NamedTemporaryFile() as _:
     args = [
-        'python',
+        'python3',
         rjsmin_path
     ]
     p = subprocess.Popen(args,
diff -up chromium-92.0.4515.107/tools/gn/bootstrap/bootstrap.py.py2 chromium-92.0.4515.107/tools/gn/bootstrap/bootstrap.py
--- chromium-92.0.4515.107/tools/gn/bootstrap/bootstrap.py.py2	2021-07-19 14:45:43.000000000 -0400
+++ chromium-92.0.4515.107/tools/gn/bootstrap/bootstrap.py	2021-07-26 17:02:23.160750472 -0400
@@ -130,7 +130,7 @@ def main(argv):
     if not options.debug:
       gn_gen_args += ' is_debug=false'
     subprocess.check_call([
-        gn_path, 'gen', out_dir,
+        gn_path, 'gen', out_dir, ' --script-executable=/usr/bin/python3',
         '--args=%s' % gn_gen_args, "--root=" + SRC_ROOT
     ])
 
