diff -up chromium-94.0.4606.71/base/allocator/allocator_shim_default_dispatch_to_partition_alloc.cc.nullptr-fix chromium-94.0.4606.71/base/allocator/allocator_shim_default_dispatch_to_partition_alloc.cc
--- chromium-94.0.4606.71/base/allocator/allocator_shim_default_dispatch_to_partition_alloc.cc.nullptr-fix	2021-10-06 15:23:11.590602162 -0400
+++ chromium-94.0.4606.71/base/allocator/allocator_shim_default_dispatch_to_partition_alloc.cc	2021-10-06 15:29:21.508762249 -0400
@@ -380,7 +380,10 @@ void PartitionFree(const AllocatorDispat
 size_t PartitionGetSizeEstimate(const AllocatorDispatch*,
                                 void* address,
                                 void* context) {
-  PA_DCHECK(address);
+  // This is used to implement malloc_usable_size(3). Per its man page, "if ptr
+  // is NULL, 0 is returned".
+  if (!address)
+    return 0;
 
 #if defined(OS_APPLE)
   if (!base::IsManagedByPartitionAlloc(address)) {
