diff --git a/net/third_party/quiche/src/http2/adapter/window_manager.h b/net/third_party/quiche/src/http2/adapter/window_manager.h
--- a/net/third_party/quiche/src/http2/adapter/window_manager.h
+++ b/net/third_party/quiche/src/http2/adapter/window_manager.h
@@ -2,6 +2,7 @@
 #define QUICHE_HTTP2_ADAPTER_WINDOW_MANAGER_H_

 #include <functional>
+#include <stddef.h>

 #include "common/platform/api/quiche_export.h"
