--- a/components/signin/internal/identity_manager/primary_account_manager.cc
+++ b/components/signin/internal/identity_manager/primary_account_manager.cc
@@ -170,7 +170,7 @@ void PrimaryAccountManager::SetSyncPrimaryAccountInternal(
     std::string pref_account_id =
         client_->GetPrefs()->GetString(prefs::kGoogleServicesAccountId);
     bool consented_to_sync =
-        client_->GetPrefs()->GetBoolean(prefs::kGoogleServicesConsentedToSync);
+        false;
 
     DCHECK(pref_account_id.empty() || !consented_to_sync ||
            pref_account_id == account_info.account_id.ToString())
