diff -up chromium-90.0.4430.85/mojo/core/channel_linux.cc.epel7-kcmp chromium-90.0.4430.85/mojo/core/channel_linux.cc
--- chromium-90.0.4430.85/mojo/core/channel_linux.cc.epel7-kcmp	2021-04-27 11:42:46.207245608 -0400
+++ chromium-90.0.4430.85/mojo/core/channel_linux.cc	2021-04-27 11:43:21.941841419 -0400
@@ -43,6 +43,25 @@
 #define EFD_ZERO_ON_WAKE O_NOFOLLOW
 #endif
 
+#ifndef F_LINUX_SPECIFIC_BASE
+#define F_LINUX_SPECIFIC_BASE 1024
+#endif
+#ifndef F_SEAL_SEAL
+#define F_SEAL_SEAL 0x0001
+#endif
+#ifndef F_SEAL_SHRINK
+#define F_SEAL_SHRINK 0x0002
+#endif
+#ifndef F_SEAL_GROW
+#define F_SEAL_GROW 0x0004
+#endif
+#ifndef F_ADD_SEALS
+#define F_ADD_SEALS (F_LINUX_SPECIFIC_BASE + 9)
+#endif
+#ifndef F_GET_SEALS
+#define F_GET_SEALS (F_LINUX_SPECIFIC_BASE + 10)
+#endif
+
 namespace mojo {
 namespace core {
 
diff -up chromium-90.0.4430.85/sandbox/policy/linux/bpf_cros_amd_gpu_policy_linux.cc.epel7-kcmp chromium-90.0.4430.85/sandbox/policy/linux/bpf_cros_amd_gpu_policy_linux.cc
--- chromium-90.0.4430.85/sandbox/policy/linux/bpf_cros_amd_gpu_policy_linux.cc.epel7-kcmp	2021-04-20 14:58:36.000000000 -0400
+++ chromium-90.0.4430.85/sandbox/policy/linux/bpf_cros_amd_gpu_policy_linux.cc	2021-04-26 11:13:36.743346891 -0400
@@ -6,7 +6,24 @@
 
 #include <errno.h>
 #include <fcntl.h>
-#include <linux/kcmp.h>
+#ifndef KCMP_FILE
+#define KCMP_FILE 0
+#endif
+#ifndef F_LINUX_SPECIFIC_BASE
+#define F_LINUX_SPECIFIC_BASE 1024
+#endif
+#ifndef F_SEAL_SEAL
+#define F_SEAL_SEAL 0x0001
+#endif
+#ifndef F_SEAL_SHRINK
+#define F_SEAL_SHRINK 0x0002
+#endif
+#ifndef F_SEAL_GROW
+#define F_SEAL_GROW 0x0004
+#endif
+#ifndef F_ADD_SEALS
+#define F_ADD_SEALS (F_LINUX_SPECIFIC_BASE + 9)
+#endif
 #include <sys/socket.h>
 
 // Some arch's (arm64 for instance) unistd.h don't pull in symbols used here
diff -up chromium-90.0.4430.85/sandbox/policy/linux/bpf_gpu_policy_linux.cc.epel7-kcmp chromium-90.0.4430.85/sandbox/policy/linux/bpf_gpu_policy_linux.cc
--- chromium-90.0.4430.85/sandbox/policy/linux/bpf_gpu_policy_linux.cc.epel7-kcmp	2021-04-20 14:58:36.000000000 -0400
+++ chromium-90.0.4430.85/sandbox/policy/linux/bpf_gpu_policy_linux.cc	2021-04-26 11:13:36.744346907 -0400
@@ -23,6 +23,22 @@
 #include "sandbox/policy/linux/sandbox_linux.h"
 #include "sandbox/policy/linux/sandbox_seccomp_bpf_linux.h"
 
+#ifndef F_LINUX_SPECIFIC_BASE
+#define F_LINUX_SPECIFIC_BASE 1024
+#endif
+#ifndef F_SEAL_SEAL
+#define F_SEAL_SEAL 0x0001
+#endif
+#ifndef F_SEAL_SHRINK
+#define F_SEAL_SHRINK 0x0002
+#endif
+#ifndef F_SEAL_GROW
+#define F_SEAL_GROW 0x0004
+#endif
+#ifndef F_ADD_SEALS
+#define F_ADD_SEALS (F_LINUX_SPECIFIC_BASE + 9)
+#endif
+
 using sandbox::bpf_dsl::Allow;
 using sandbox::bpf_dsl::Arg;
 using sandbox::bpf_dsl::Error;
diff -up chromium-90.0.4430.93/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc.epel7-kcmp chromium-90.0.4430.93/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc
--- chromium-90.0.4430.93/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc.epel7-kcmp	2021-04-27 16:08:22.416365113 -0400
+++ chromium-90.0.4430.93/sandbox/linux/seccomp-bpf-helpers/syscall_parameters_restrictions.cc	2021-04-27 16:08:43.571725851 -0400
@@ -57,6 +57,25 @@
 
 #endif  // defined(OS_ANDROID)
 
+#ifndef F_LINUX_SPECIFIC_BASE
+#define F_LINUX_SPECIFIC_BASE 1024
+#endif
+#ifndef F_SEAL_SEAL
+#define F_SEAL_SEAL 0x0001
+#endif
+#ifndef F_SEAL_SHRINK
+#define F_SEAL_SHRINK 0x0002
+#endif
+#ifndef F_SEAL_GROW
+#define F_SEAL_GROW 0x0004
+#endif
+#ifndef F_ADD_SEALS
+#define F_ADD_SEALS (F_LINUX_SPECIFIC_BASE + 9)
+#endif
+#ifndef F_GET_SEALS
+#define F_GET_SEALS (F_LINUX_SPECIFIC_BASE + 10)
+#endif
+
 #if defined(__arm__) && !defined(MAP_STACK)
 #define MAP_STACK 0x20000  // Daisy build environment has old headers.
 #endif
diff -up chromium-90.0.4430.93/ui/events/ozone/evdev/event_converter_evdev_impl.cc.epel7-kcmp chromium-90.0.4430.93/ui/events/ozone/evdev/event_converter_evdev_impl.cc
--- chromium-90.0.4430.93/ui/events/ozone/evdev/event_converter_evdev_impl.cc.epel7-kcmp	2021-04-28 09:55:12.941124592 -0400
+++ chromium-90.0.4430.93/ui/events/ozone/evdev/event_converter_evdev_impl.cc	2021-04-28 09:55:42.071601437 -0400
@@ -17,6 +17,10 @@
 #include "ui/events/ozone/evdev/device_event_dispatcher_evdev.h"
 #include "ui/events/ozone/evdev/event_device_util.h"

+#ifndef SW_PEN_INSERTED
+#define SW_PEN_INSERTED                0x0f  /* set = pen inserted */
+#endif
+
 namespace ui {
 
 namespace {
