diff -up chromium-84.0.4147.105/tools/gn/src/gn/err.h.gn-gcc-cleanup chromium-84.0.4147.105/tools/gn/src/gn/err.h
--- chromium-84.0.4147.105/tools/gn/src/gn/err.h.gn-gcc-cleanup	2020-08-01 10:04:57.354719575 -0400
+++ chromium-84.0.4147.105/tools/gn/src/gn/err.h	2020-08-01 10:04:51.653974728 -0400
@@ -55,7 +55,7 @@ class Err {
       const std::string& help_text = std::string());
 
   Err(const Err& other);
-
+  Err& operator=(const Err& other) = default;
   ~Err();
 
   bool has_error() const { return has_error_; }
diff -up chromium-84.0.4147.105/tools/gn/src/gn/label_pattern.h.gn-gcc-cleanup chromium-84.0.4147.105/tools/gn/src/gn/label_pattern.h
--- chromium-84.0.4147.105/tools/gn/src/gn/label_pattern.h.gn-gcc-cleanup	2020-08-01 10:24:18.405934036 -0400
+++ chromium-84.0.4147.105/tools/gn/src/gn/label_pattern.h	2020-08-01 10:24:56.664265755 -0400
@@ -33,6 +33,7 @@ class LabelPattern {
                const std::string_view& name,
                const Label& toolchain_label);
   LabelPattern(const LabelPattern& other);
+  LabelPattern& operator=(const LabelPattern& other) = default;
   ~LabelPattern();
 
   // Converts the given input string to a pattern. This does special stuff
diff -up chromium-84.0.4147.105/tools/gn/src/gn/substitution_list.h.gn-gcc-cleanup chromium-84.0.4147.105/tools/gn/src/gn/substitution_list.h
--- chromium-84.0.4147.105/tools/gn/src/gn/substitution_list.h.gn-gcc-cleanup	2020-08-01 10:04:51.721971684 -0400
+++ chromium-84.0.4147.105/tools/gn/src/gn/substitution_list.h	2020-08-01 10:04:51.765969715 -0400
@@ -15,6 +15,7 @@ class SubstitutionList {
  public:
   SubstitutionList();
   SubstitutionList(const SubstitutionList& other);
+  SubstitutionList& operator=(const SubstitutionList& other) = default;
   ~SubstitutionList();
 
   bool Parse(const Value& value, Err* err);
diff -up chromium-84.0.4147.105/tools/gn/src/gn/substitution_pattern.h.gn-gcc-cleanup chromium-84.0.4147.105/tools/gn/src/gn/substitution_pattern.h
--- chromium-84.0.4147.105/tools/gn/src/gn/substitution_pattern.h.gn-gcc-cleanup	2020-08-01 10:04:51.637975444 -0400
+++ chromium-84.0.4147.105/tools/gn/src/gn/substitution_pattern.h	2020-08-01 10:04:57.305721767 -0400
@@ -35,6 +35,7 @@ class SubstitutionPattern {
 
   SubstitutionPattern();
   SubstitutionPattern(const SubstitutionPattern& other);
+  SubstitutionPattern& operator=(const SubstitutionPattern& other) = default;
   ~SubstitutionPattern();
 
   // Parses the given string and fills in the pattern. The pattern must only
