diff -up chromium-89.0.4389.82/chrome/browser/component_updater/registration.cc.widevine-no-download chromium-89.0.4389.82/chrome/browser/component_updater/registration.cc
--- chromium-89.0.4389.82/chrome/browser/component_updater/registration.cc.widevine-no-download	2021-03-18 14:10:27.606759188 -0400
+++ chromium-89.0.4389.82/chrome/browser/component_updater/registration.cc	2021-03-18 14:10:41.792833905 -0400
@@ -96,10 +96,6 @@ void RegisterComponentsForUpdate(bool is
   // TODO(crbug.com/1069814): Remove after 2021-10-01.
   CleanUpPepperFlashComponent();
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-  RegisterWidevineCdmComponent(cus);
-#endif  // BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-
 #if BUILDFLAG(ENABLE_NACL) && !defined(OS_ANDROID)
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   // PNaCl on Chrome OS is on rootfs and there is no need to download it. But
