diff -up chromium-96.0.4664.110/content/public/common/content_features.cc.enable-WebRTCPipeWireCapturer-byDefault chromium-96.0.4664.110/content/public/common/content_features.cc
--- chromium-96.0.4664.110/content/public/common/content_features.cc.enable-WebRTCPipeWireCapturer-byDefault	2021-12-20 11:24:28.862940185 -0500
+++ chromium-96.0.4664.110/content/public/common/content_features.cc	2021-12-20 11:24:43.397027114 -0500
@@ -1125,7 +1125,7 @@ const base::Feature kAuraWindowSubtreeCa
 // Controls whether the PipeWire support for screen capturing is enabled on the
 // Wayland display server.
 const base::Feature kWebRtcPipeWireCapturer{"WebRTCPipeWireCapturer",
-                                            base::FEATURE_DISABLED_BY_DEFAULT};
+                                            base::FEATURE_ENABLED_BY_DEFAULT};
 #endif  // defined(WEBRTC_USE_PIPEWIRE)
 
 enum class VideoCaptureServiceConfiguration {
