diff -up chromium-94.0.4606.61/remoting/host/mojom/remoting_mojom_traits.h.extra-qualification chromium-94.0.4606.61/remoting/host/mojom/remoting_mojom_traits.h
--- chromium-94.0.4606.61/remoting/host/mojom/remoting_mojom_traits.h.extra-qualification	2021-09-25 01:48:28.812426004 +0000
+++ chromium-94.0.4606.61/remoting/host/mojom/remoting_mojom_traits.h	2021-09-25 01:48:59.072267580 +0000
@@ -17,7 +17,7 @@
 namespace mojo {
 
 template <>
-class mojo::StructTraits<remoting::mojom::ClipboardEventDataView,
+class StructTraits<remoting::mojom::ClipboardEventDataView,
                          ::remoting::protocol::ClipboardEvent> {
  public:
   static const std::string& mime_type(
