diff -up chromium-91.0.4472.77/base/test/BUILD.gn.nofontconfigcache chromium-91.0.4472.77/base/test/BUILD.gn
--- chromium-91.0.4472.77/base/test/BUILD.gn.nofontconfigcache	2021-06-01 16:41:40.094756454 -0400
+++ chromium-91.0.4472.77/base/test/BUILD.gn	2021-06-01 16:42:47.736100516 -0400
@@ -198,9 +198,6 @@ static_library("test_support") {
     sources += [ "test_file_util_linux.cc" ]
     public_deps += [ ":fontconfig_util_linux" ]
     data_deps += [ "//third_party/test_fonts" ]
-    if (current_toolchain == host_toolchain) {
-      data_deps += [ ":do_generate_fontconfig_caches" ]
-    }
   }
 
   if (is_mac) {
