diff -up chromium-94.0.4606.71/remoting/codec/codec_test.cc.missing-header chromium-94.0.4606.71/remoting/codec/codec_test.cc
--- chromium-94.0.4606.71/remoting/codec/codec_test.cc.missing-header	2021-10-07 18:01:11.846157616 +0000
+++ chromium-94.0.4606.71/remoting/codec/codec_test.cc	2021-10-07 18:01:22.913662166 +0000
@@ -8,6 +8,7 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+#include <cmath>
 #include <memory>
 #include <utility>
 
