diff -up chromium-93.0.4577.63/chrome/browser/component_updater/registration.cc.widevine-no-download chromium-93.0.4577.63/chrome/browser/component_updater/registration.cc
--- chromium-93.0.4577.63/chrome/browser/component_updater/registration.cc.widevine-no-download	2021-09-02 08:37:03.910692041 -0400
+++ chromium-93.0.4577.63/chrome/browser/component_updater/registration.cc	2021-09-02 08:45:07.446778150 -0400
@@ -118,10 +118,6 @@ void RegisterComponentsForUpdate(bool is
   RegisterMediaFoundationWidevineCdmComponent(cus);
 #endif
 
-#if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-  RegisterWidevineCdmComponent(cus);
-#endif  // BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-
 #if BUILDFLAG(ENABLE_NACL) && !defined(OS_ANDROID)
 #if BUILDFLAG(IS_CHROMEOS_ASH)
   // PNaCl on Chrome OS is on rootfs and there is no need to download it. But
